db/migrate/20200601141555_create_library_locations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLibraryLocations < ActiveRecord::Migration[6.0]
+  def self.up
+    create_table :library_locations do |t|
+      t.column :name, :string
+      t.column :address, :string
+      t.column :established, :integer
+    end
+  end
+
+  def self.down
+    drop_table :library_locations
+  end
```

Comment:
Just so you know on all of these `def self.up`/`def self.down` you can replace it with the more modern `def change` that Rails has. With that Rails will automatically know how to reverse the migration for you